webrtc.html

```diff
@@ -9942,14 +9942,14 @@ <h3>Mandatory To Implement Stats</h3>
         inherited dictionaries, and also attributes packetsReceived,
```

Comment:
Above this point: RTCRTPStreamStats mediaType has been renamed to kind
RTCRTPStreamStats.trackId has moved to RTCInboundRtpStreamStats and RTCOutboundRtpStreamStats

RTCReceivedRTPStreamStats doesn't have bytesReceived - it's moved to RTCInboundRtpStreamStats